group_vars/elasticsearch/public.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+# ELASTICSEARCH ###############################################################
+
+elasticsearch_repo: "deb http://packages.elastic.co/elasticsearch/1.5/debian stable main"
+elasticsearch_package: "elasticsearch=1.5.2"
+elasticsearch_network_host: 0.0.0.0
+elasticsearch_http_port: 9200
+elasticsearch_script_inline: false
+elasticsearch_script_indexed: false
+
+# CONSUL ######################################################################
+
+consul_service_config:
+  service:
```

Comment:
You've done a much more complete job than I intended for this ticket.  We actually only need the service definitions for Prometheus, and Prometheus will only monitor the exporters, not the services itself.  However, having service definitions for everything will enable lots of use cases we haven't planned yet, so it's nice to see this.

Reply:
Thanks a lot!